source/includes/steps-fle-convert-to-a-remote-master-key-azure.yaml

```diff
@@ -0,0 +1,235 @@
+  requests you make from the client application to create and store a
+  new data encryption key:
+
+  .. image:: /figures/CSFLE_Data_Key_KMS.png
+     :alt: Diagram that describes creating a data encryption key when using a KMS provider
+
+  Provide your client with the following information to access the master key:
+
+  .. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Field
+     - Required
+     - Description
+
+   * - keyName
+     - Yes
+     - Name of the master key
+
+   * - keyVersion
+     - No
+     - Version of the master key
+
+   * - keyVaultEndpoint
+     - Yes
+     - URL of the key vault. E.g. myVaultName.vault.azure.net
+
+  Once you have the required information, run the following code to
+  generate the new data encryption key:
+
+  .. tabs-drivers::
+
+     .. tab::
+        :tabid: java-sync
+
+        .. code-block:: Java
+
+           // TODO: update for Azure
+           ClientEncryption clientEncryption = ClientEncryptions.create(ClientEncryptionSettings.builder()
+               .keyVaultMongoClientSettings(MongoClientSettings.builder()
+                   .applyConnectionString(new ConnectionString("mongodb://localhost:27017"))
+                   .build())
+               .keyVaultNamespace(keyVaultNamespace)
+               .kmsProviders(kmsProviders)
+               .build());
+
+           BsonString masterKeyRegion = new BsonString("<Master Key AWS Region>"); // e.g. "us-east-2"
+           BsonString masterKeyArn = new BsonString("<Master Key ARN>"); // e.g. "arn:aws:kms:us-east-2:111122223333:alias/test-key"
+           DataKeyOptions dataKeyOptions = new DataKeyOptions().masterKey(
+               new BsonDocument()
+                   .append("region", masterKeyRegion)
+                   .append("key", masterKeyArn));
+
+           BsonBinary dataKeyId = clientEncryption.createDataKey("aws", dataKeyOptions);
+           String base64DataKeyId = Base64.getEncoder().encodeToString(dataKeyId.getData());
+
+           System.out.println("DataKeyId [base64]: " + base64DataKeyId);
+
+     .. tab::
+        :tabid: nodejs
+
+        .. code-block:: javascript
+
+           // TODO: update for Azure
+           const encryption = new ClientEncryption(client, {
+               keyVaultNamespace,
+               kmsProviders
+           });
+           const key = await encryption.createDataKey('aws', {
+              masterKey: {
+                key: '<Master Key ARN>', // e.g. 'arn:aws:kms:us-east-2:111122223333:alias/test-key'
+              }
+           });
+
+           const base64DataKeyId = key.toString('base64');
+           console.log('DataKeyId [base64]: ', base64DataKeyId);
+
+     .. tab::
+        :tabid: python
+
+        In ``app.py``:
+
+        .. code-block:: python
+
+           master_key = {
+             "keyName": "<Azure key name>",
+             # "keyVersion": "<Azure key vault endpoint>",
+             "keyVaultEndpoint": "<Azure key version>"
```

Comment:
These seem to be swapped. It should be:
```
"keyVersion": "<Azure key version>",
"keyVaultEndpoint": "<Azure key vault endpoint>"
```

Reply:
Whoops, thanks for spotting this, will update.